### ISSUE TITLE
508-defect-2 [COGNITION, SCREEN READER]: View dependents modal must reference its `h1` using `aria-labelledby` or be appropriately labelled

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#41108]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I expect modal dialogs to be labelled so their purpose can be understandable to me.

## Details

Currently, the modal dialog uses the `aria-labelledby` attribute to reference what _should_ be the first heading in the modal. However, the first heading in the modal does not have the respective `id` attribute needed to connect the two together.

## Acceptance Criteria

- [ ] The first heading "Please make sure..."'s `ID` = "dependency-verification-title"

## WCAG or Vendor Guidance (optional)
[ARIA dialog and alertdialog must have an accessible name](https://dequeuniversity.com/rules/axe/4.4/aria-dialog-name?application=AxeChrome)

### Before
<img width="1792" alt="Screen Shot 2022-05-09 at 4 41 23 PM" src="https://user-images.githubusercontent.com/14154792/167497355-d6178360-dc8e-444b-bbb6-c610a6a9441c.png">

### After
<img width="1150" alt="Screen Shot 2022-05-09 at 4 54 58 PM" src="https://user-images.githubusercontent.com/14154792/167497370-b05b11a2-cd26-41e9-8ad2-8d0bcd90c3fb.png">

**Comment by joshkimux** [comment]
@Mottie Hey Robin! I don't think this is in the scope of our team anymore right?
@artsymartha68 by any chance do you know who might be the new owners of this team? Or maybe a place I can ask to find out?

**Comment by Mottie** [comment]
🙅🏽 This app is obsolete

**Comment by tblackwe** [comment]
@Mottie can you let me know more about your comment?

> 🙅🏽 This app is obsolete

Does this work not need to be picked up?

**Comment by Mottie** [comment]
Sorry, I misspoke. The app still exists in production. The MODAL within the app isn't being worked on.

Some history: after looking at the dependency verification app MODAL, we determined that it would only check if the dependents were updated _if_ the Veteran visited that app, so we contacted the va-notify team to email Veterans to verify their dependents periodically.

**Comment by tblackwe** [comment]
@ohgreatkate: I've asked @zack to review this ticket on our side.

**Comment by sortizsh** [comment]
@evansmith can you check in with Robin to figure out if we should make this fix? if so, make it! ty

**Comment by evansmith** [comment]
@steele-lm @sortizsh @tblackwe I think that we can close this one. With the combination of the outdated nature of this section combined with the technical issues of aria-label within VaModal for VoiceOver to use (a modalTitle attribute is needed to provide context for the modal in VoiceOver but that introduces a new text element that we don't want).
